src/modules/generator/main_impl.h

```diff
@@ -288,10 +288,8 @@ int secp256k1_pedersen_commitment_parse(const secp256k1_context* ctx, secp256k1_
         !secp256k1_ge_set_xquad(&ge, &x)) {
```

Comment:
```suggestion
        !secp256k1_ge_x_on_curve_var(&x)) {
```
and we can drop `ge` entirely.

Reply:
Force-pushed to do this.